app/models/pom_caseload.rb

```diff
@@ -32,25 +32,40 @@ def load_allocations
     offender_ids = allocation_list.map(&:nomis_offender_id)
     offenders = OffenderService.get_multiple_offenders(offender_ids)
 
+    # Lookup responsibility overrides and store them in a hash for this
```

Comment:
eek - don't think we should be jumping through this many hoops just to do something really simple. A responsibility should be attached somewhere in the rails model (sorry probably me forgot to do it) and we should just go and get it. Then use the presenter to display it if it's there, rather than writing all this code. We need to push towards a DRY and simple approach.

Reply:
👍 
Was thinking about moving it to the offender#pom_responsibility method, but I guess the presenter is a good place.